src/main/resources/META-INF/plugin.xml

```diff
@@ -8,14 +8,15 @@
     <depends>org.jetbrains.plugins.yaml</depends>
     <depends>com.intellij.modules.json</depends>
     <depends>org.jetbrains.plugins.terminal</depends>
-    <depends>com.jetbrains.sh</depends>
     <resource-bundle>messages.GitlabCIAid</resource-bundle>
 
+    <depends optional="true" config-file="plugin-shell.xml">com.jetbrains.sh</depends>
```

Comment:
optional dependencies to allow installing without any of them and to use them as soon as available.